interface/usergroup/mfa_registrations.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Multi-Factor Authentication Management
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2018 Rod Roark <rod@sunsetsystems.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE CNU General Public License 3
+ */
+
+require_once('../globals.php');
+
+use OpenEMR\Core\Header;
+
+function writeRow($method, $name)
+{
+    echo "        <tr><td>&nbsp;";
+    echo text($method);
+    echo "&nbsp;</td><td>&nbsp;";
+    echo text($name);
+    echo "&nbsp;</td><td>";
+    echo "<input type='button' onclick='delclick(\"" . attr($method) . "\", \"" .
+        attr($name) . "\")' value='" . xla('Delete') . "' />";
+    echo "</td></tr>\n";
+}
+
+$userid = $_SESSION['authId'];
+
+$message = '';
+if (!empty($_POST['form_delete_method'])) {
```

Comment:
Place the following code block here (just trust me :) ):
```
    if (!verifyCsrfToken($_POST["csrf_token_form"])) {
        die(xlt('Authentication Error'));
    }
```
(this is for: https://www.open-emr.org/wiki/index.php/Codebase_Security#CSRF and will pair with the next comment i write )